mitosheet/css/taskpanes/Graph/GraphSidebar.css

```diff
@@ -52,6 +52,25 @@
     box-sizing: border-box;
 }
 
+.gtitle-highlighted .g-gtitle {
+    outline: 1px solid var(--mito-text-light);
+}
+
+.xtitle-highlighted .g-xtitle {
+    outline: 1px solid var(--mito-text-light);
```

Comment:
Can we try to make this look a bit nicer? Probably:

1. Some padding around the text?
2. Darker lines? 

Maybe make a little mockup for it and then implement, I trust your design eye

<img width="199" alt="Screenshot 2024-01-25 at 2 32 54 PM" src="https://github.com/mito-ds/mito/assets/18709905/0512e8f4-598a-4ca5-af81-319d1fd310c5">

Reply:
Agh yeah this is tricky. For some reason any css that I added to this element was being ignored. We can adjust the color of the line but it might be a bigger thing to add padding / border radius unfortunately

Reply:
Okay good callout. Let's leave for now then.